### PR TITLE
[Improvement] Made one minor change.

### PR DESCRIPTION
**Pull request by OXYMORON16**
All the links on http://openwisp.io/docs/developer/contributing.html ; opens on the same tab which is troublesome for some users. So, I added some javascript so that the links opens in the new tab.
plus: I corrected a spelling mistake i noticed.

### CODE REVIEW

File: _templates/layout.html [hunk 1]
```diff
@@ -19,4 +19,11 @@
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');	
 </script>
+<script type="text/javascript">
```
Comment (atb00ker) [inline _templates/layout.html]: I don't think you need to do this.
Simply put this in the head section: `<base target="_blank">` 
If there is a specific URL that shouldn't open on a different tab, add `_self` to it.  :-)

Comment (OXYMORON16) [reply _templates/layout.html]: Whoa, that was genius!!!

File: _templates/layout.html [hunk 2]
```diff
@@ -19,4 +20,5 @@
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');	
 </script>
+
```
Comment (atb00ker) [inline _templates/layout.html]: Hi, please remove this blank space change, it's not required now!

File: _templates/layout.html [hunk 3]
```diff
@@ -19,4 +20,8 @@
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');	
 </script>
+<<<<<<< HEAD
```
Comment (dwang) [inline _templates/layout.html]: These lines (23-26) should not be committed. Can you remove them?

**Pull request closed without merging**